apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -147,6 +147,10 @@ data:
 {{ toYaml .Values.api.services.metrics.prometheus | indent 10 }}
         {{- end }}
       {{- end }}
+      {{- if .Values.api.services.subscription.enabled }}
+      subscription:
+        {{- toYaml .Values.api.services.subscription | nindent 8 }}
```

Comment:
nerror ;)

Reply:
`nindent` is almost identical to `indent`, but begins a new line. I used this one with the `{{- ... }}` syntax